crates/egui_extras/src/datepicker/button.rs

```diff
@@ -39,8 +39,16 @@ impl<'a> DatePickerButton<'a> {
     /// Add id source.
     /// Must be set if multiple date picker buttons are in the same Ui.
     #[inline]
-    pub fn id_source(mut self, id_source: &'a str) -> Self {
-        self.id_source = Some(id_source);
+    #[deprecated = "Renamed id_salt"]
+    pub fn id_source(self, id_salt: &'a str) -> Self {
+        self.id_salt(id_salt)
+    }
```

Comment:
nitpick: I prefer putting the non-deprecated functions above the deprecated ones, because the order of them is reflected in the order in the docs